sdk/core/azure-core/src/main/java/com/azure/core/http/policy/DefaultRedirectStrategy.java

```diff
@@ -160,7 +166,11 @@ private boolean alreadyAttemptedRedirectUrl(String redirectUrl,
      */
     private boolean isValidRedirectCount(int tryCount) {
         if (tryCount >= getMaxAttempts()) {
-            LOGGER.error("Request has been redirected more than {} times.", getMaxAttempts());
+            LOGGER.atError()
+                .addKeyValue(LoggingKeys.TRY_COUNT_KEY, tryCount)
```

Comment:
Does this need to include the try count as that is inferred by attempting more than the maximum limit